internal/api/filter_tests.go

```diff
@@ -14,8 +14,7 @@ type FilterTestsParams struct {
 }
 
 type FilteredTest struct {
-	Path   string `json:"path"`
-	Reason string `json:"reason"`
```

Comment:
I have removed the `Reason` for now because we don't use it on the client side, so the API response is not restricted to having `reason` with `string` value.

Reply:
You can use `omitempty`, to make things line up?

Reply:
I can, but I'm thinking to change the type from string to an array of string, as 1 file can be slow and has quarantined. And because we don't need it on the client side, I don't see a reason why we need to parse it into the struct.

Reply:
All good!